jobs/periodic/redact-inactive-accounts/index.ts

```diff
@@ -49,7 +49,7 @@ export default async function execute() {
     const screenersToBeRedacted = await prisma.screener.findMany({
         where: {
             active: false,
-            updatedAt: {
+            lastLogin: {
```

Comment:
What if lastLogin is NULL?

Reply:
I had the same question and based on my test it should not make a difference.
Nevertheless, I'll verify it again just to be sure.

Reply:
Yes, it's ignoring null values.
I've added an integration test to make sure that this is also the case in the future 🙂

Reply:
(we could create a backlog item for this to be done in 2027 :))

Reply:
😅 tbh. there are none in production as there is a default `now()`.